cmd/run.go

```diff
@@ -226,6 +226,11 @@ func SetupRunFlags(cmd *cobra.Command) {
 		"from an empty string to a non-empty string (or from a non-empty string to the empty string) requires a resync.")
 	cmd.PersistentFlags().Uint("state-syncer-mempool-txn-sync-limit", 10000, "The maximum number of transactions to "+
 		"process in the mempool tx state syncer at a time.")
+
+	// PoS Checkpoint Syncing
+	// TODO: a default value is a little tricky here since we don't know what network we're using.
+	cmd.PersistentFlags().StringSlice("checkpoint-syncing-providers", []string{}, "A comma-separated list of URLs that "+
+		"support the committed tip block info endpoint to be used for checkpoint syncing.")
```

Comment:
```suggestion
	// PoS Checkpoint Syncing
	// TODO: a default value is a little tricky here since we don't know what network we're using.
	cmd.PersistentFlags().StringSlice(fmt.Sprintf("checkpoint-syncing-providers", []string{}, "A comma-separated list of URLs that "+
		"supports the committed tip block info endpoint to be used for checkpoint syncing. "+
		"If unset, the field will default to %v on mainnet and %v on testnet", DefaultMainnetCheckpointProvider, DefaultTestnetCheckpointProvider))
```